adapters/aws-sns.md

```diff
@@ -0,0 +1,25 @@
+# AWS Simple Notification Service CloudEvents Adapter
+
+This document describes how to convert
+[AWS SNS notifications](https://docs.aws.amazon.com/sns/latest/api/Welcome.html)
+into a CloudEvents.
+
+AWS SNS notification documentation:
+https://docs.aws.amazon.com/sns/latest/api/Welcome.html
+
+All SNS notifications are converted into CloudEvents using the
+same pattern as described in the following table:
+
+| CloudEvents Attribute | Value                                           |
+| :-------------------- | :---------------------------------------------- |
+| `id`                  | "x-amz-sns-message-id" value                    |
+| `source`              | "x-amz-sns-topic-arn" value                     |
+| `specversion`         | `0.3-wip`                                       |
+| `type`                | `aws.com/sns/` + "x-amz-sns-message-type" value |
```

Comment:
Why `aws.com` here? Seems like the actual domain is `aws.amazonaws.com`.

Reply:
yep - I'll fix - thanks!  and do reverse-dns too :-)